assets/base.css

```diff
@@ -622,7 +622,8 @@ h6:empty {
   margin-left: 1rem;
 }
 
-.link[role="link"]:not([href]) {
+.link[role="link"]:not([href]),
+a:not([href]) {
```

Comment:
```suggestion
a:not([href]) {
```

I'm wondering if we even need `.link[role="link"]:not([href]),` if we've got `a:not([href])`.

Reply:
Since `a:not([href])` already takes care of all "disabled" anchor links, I'm good with removing `.link[role="link"]:not([href]),` entirely. I just finished testing without `.link[role="link"]:not([href]),` and everything checks out.

Reply:
😆 the amount of discussion this seemingly benign selector has warranted over the span of ~3 PRs. So funny.

As discussed, I feel `a:not([href])` is a little broad to change the cursor. Including the `[role="link"]` makes it a little more intentional and also preserves the original intent of creating a visual cue that might alert us to empty links that are _missing_ the needed `role`. I don't see where the slight increase in specificity would matter for this selector. 

I also cannot remember the context behind using `.link` but it's likely it was an interim "safe scoping" decision at some point so I'm fine removing that if we keep the attr selector.

Reply:
**Update**: One thing I noticed when using:

```css
a[role="link"]:not([href]) {
  cursor: not-allowed;
}
```

The buttons from the following sections are using the above CSS rule as it overrides `.button[aria-disabled='true']`:

- [Rich text section > button](https://user-images.githubusercontent.com/28404165/153301443-8cb58be5-9c0e-4983-8c44-297d499c7029.png)
- [Image banner > buttons](https://user-images.githubusercontent.com/28404165/153301634-111d79c2-2e7e-4cb4-91e8-eb27232b47f4.png)
- [Slideshow > buttons](https://user-images.githubusercontent.com/28404165/153301744-9b80e7d4-4016-4189-b169-78056f5ccbc7.png)

And here's what it looks like when using the `a:not([href])` selector:

![image](https://user-images.githubusercontent.com/28404165/153302336-c9f4922b-094d-47bc-9bfe-969a1ac50f9b.png)

Reply:
I didn't realize this was such a can of worms!  Thanks for all the context @kmeleta .  Indeed, this benign selector has a lot of sway in PR conversations, it seems. 🤣